### DISCUSSION TITLE
ChatGPT Writer Extension not opening in GMail Response

### DISCUSSION THREAD

**Discussion by mdohadwala-clgx** [GorvGoyl/ChatGPT-Writer#107] [Report Bugs and Issues]
ChatGPT writer is not opening in GMail response editor. It is asking me to log into the open.ai. I was logged into it but still it was not working. Request you to please advise what is the issue with the extension.;

**Comment by GorvGoyl** [comment]
Hello, The recent ChatGPT update caused some issues with the extension's
performance, but these have now been fixed.
